From 4a077b0c62ed5dec0207d2c0b5737621b989105a Mon Sep 17 00:00:00 2001
From: Alvaro Guzman <alvaro.guzman@owasys.com>
Date: Wed, 31 Aug 2022 13:50:26 +0200
Subject: [PATCH] Downgraded ddr trainning files version. In BalenaOS the
 meta-imx recipe that provided ddr files 202006 is incompatible with Dunfell
 balena recipes

---
 iMX8M/soc.mak | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/iMX8M/soc.mak b/iMX8M/soc.mak
index f6e236e..c90fe0f 100644
--- a/iMX8M/soc.mak
+++ b/iMX8M/soc.mak
@@ -61,8 +61,8 @@ VAL_BOARD = val
 QSPI_HEADER = ../scripts/fspi_header
 QSPI_PACKER = ../scripts/fspi_packer.sh
 VERSION = v2
-LPDDR_FW_VERSION = _202006
-DDR_FW_VERSION = _202006
+LPDDR_FW_VERSION = _201904
+DDR_FW_VERSION = _201904
 else
 PLAT = imx8mq
 HDMI = yes
-- 
2.25.1

